dice_ml/explainer_interfaces/dice_random.py

```diff
@@ -109,11 +109,17 @@ class of query_instance for binary classification.
         cfs_df = None
         candidate_cfs = pd.DataFrame(
             np.repeat(query_instance.values, sample_size, axis=0), columns=query_instance.columns)
-        # Loop to change one feature at a time, then two features, and so on.
+        # Loop to change one feature at a time ##->(NOT TRUE), then two features, and so on.
         for num_features_to_vary in range(1, len(self.features_to_vary)+1):
+            # commented lines allow more values to change as num_features_to_vary increases, instead of .at you should use .loc
+            # is deliberately left commented out to let you choose.
+            # is slower, but more complete and still faster than genetic/KDtree
+            # selected_features = np.random.choice(self.features_to_vary, (sample_size, num_features_to_vary), replace=True)
             selected_features = np.random.choice(self.features_to_vary, (sample_size, 1), replace=True)
             for k in range(sample_size):
-                candidate_cfs.at[k, selected_features[k][0]] = random_instances.at[k, selected_features[k][0]]
+                candidate_cfs.at[k, selected_features[k][0]] = random_instances._get_value(k, selected_features[k][0])
+                # If you only want to change one feature, you should use _get_value
+                # candidate_cfs.iloc[k][selected_features[k]]=random_instances.iloc[k][selected_features[k]]
```

Comment:
@gaugup, these are the observation that I mentioned in the initial comment of the PR. Do you prefer the standard version with .get_value change or a more complete version(commented) with increasing substitution of random features?